*Replace this text with your own name and a brief description of what this notebook does.*

## Assignment 6-1

This assignment will use the MoMA artwork dataset from Lab 5.

Write a piece of code that computes the range, variance, and standard deviation of the widths of the artwork in MoMA's collection.  Use the unfiltered dataset.

In [1]:
import matplotlib.pyplot as plt
import pandas as pd
%matplotlib inline

In [3]:
art = pd.read_csv("../../../data/Artworks.csv")
art.head()

Title  \
0  Ferdinandsbrücke Project, Vienna, Austria (Ele...   
1  City of Music, National Superior Conservatory ...   
2  Villa near Vienna Project, Outside Vienna, Aus...   
3  The Manhattan Transcripts Project, New York, N...   
4  Villa, project, outside Vienna, Austria, Exter...   

                     Artist ConstituentID  \
0               Otto Wagner          6210   
1  Christian de Portzamparc          7470   
2                Emil Hoppe          7605   
3           Bernard Tschumi          7056   
4                Emil Hoppe          7605   

                                   ArtistBio Nationality BeginDate EndDate  \
0                      (Austrian, 1841–1918)  (Austrian)    (1841)  (1918)   
1                        (French, born 1944)    (French)    (1944)     (0)   
2                      (Austrian, 1876–1957)  (Austrian)    (1876)  (1957)   
3  (French and Swiss, born Switzerland 1944)          ()    (1944)     (0)   
4                      (Austrian, 1876–1957)  (Austrian)    (1876)  (1957)   

   Gender  Date                                             Medium  \
0  (Male)  1896      Ink and cut-and-pasted painted pages on paper   
1  (Male)  1987                  Paint and colored pencil on print   
2  (Male)  1903  Graphite, pen, color pencil, ink, and gouache ...   
3  (Male)  1980  Photographic reproduction with colored synthet...   
4  (Male)  1903  Graphite, color pencil, ink, and gouache on tr...   

        ...                                             ThumbnailURL  \
0       ...        http://www.moma.org/media/W1siZiIsIjU5NDA1Il0s...   
1       ...        http://www.moma.org/media/W1siZiIsIjk3Il0sWyJw...   
2       ...        http://www.moma.org/media/W1siZiIsIjk4Il0sWyJw...   
3       ...        http://www.moma.org/media/W1siZiIsIjEyNCJdLFsi...   
4       ...        http://www.moma.org/media/W1siZiIsIjEyNiJdLFsi...   

  Circumference (cm) Depth (cm) Diameter (cm) Height (cm) Length (cm)  \
0                NaN        NaN           NaN     48.6000         NaN   
1                NaN        NaN           NaN     40.6401         NaN   
2                NaN        NaN           NaN     34.3000         NaN   
3                NaN        NaN           NaN     50.8000         NaN   
4                NaN        NaN           NaN     38.4000         NaN   

  Weight (kg)  Width (cm) Seat Height (cm) Duration (sec.)  
0         NaN    168.9000              NaN             NaN  
1         NaN     29.8451              NaN             NaN  
2         NaN     31.8000              NaN             NaN  
3         NaN     50.8000              NaN             NaN  
4         NaN     19.1000              NaN             NaN  

[5 rows x 29 columns]

In [4]:
width_range = art["Width (cm)"].max() - art["Width (cm)"].min()
width_range

9144.0

In [5]:
art["Width (cm)"].var()

4518.495161551289

In [6]:
art["Width (cm)"].std()

67.21975276324132

1. What are the units for the range, variance, and standard deviation that you computed above?
2. If we filter the dataset to only contain artwork with a width less than 500cm (similar to Lab 5), the variance becomes $1348.1cm^2$ and the standard deviation becomes 36.7cm.  How did the variance and standard deviation change?  Does this make sense?

Write your answers below.

1. The units are cm for the range and standard deviation, and $cm^2$ for the variance.  Thus, the range is 9144 cm, the variance is 4518.5 $cm^2$, and the standard deviation is 67.2 cm.
2. The variance and standard deviation both became smaller when we filtered the dataset.  Yes, this makes sense, because by removing the outliers, the data is less spread out around the mean, leading to a lower variance and standard deviation.